client.go

```diff
@@ -458,3 +469,13 @@ func newRestyClient(cfg Config, auth authenticator) *resty.Client {
 			return fmt.Errorf("%w: %s", ErrUnrecognizedAPIResponse, r.Body())
 		})
 }
+
+// GetXPriv retrieves the xPriv (extended private key) associated with the client.
+func (c *Client) GetXPriv() *bip32.ExtendedKey {
```

Comment:
We follow the convention of avoiding the 'Get' prefix for getter methods. Instead, we name them directly, like `XPriv` instead of `GetXPriv`.